wallet/version/helpers.go

```diff
@@ -23,7 +27,26 @@ func GetNetworkVersionThroughGRPC(hosts []string) (string, error) {
 		}
 		return version, nil
 	}
-	return "", ErrNoHealthyNodeAvailableForVersionCheck
+
+	// Before advising the users to check their network configuration, we
+	// ping vega.xyz to verify whether or not they are connected to the internet.
```

Comment:
```suggestion
	// Before advising users to check their network configuration, the software
	// pings vega.xyz to verify whether or not they are connected to the internet.
```
Suggested rewording, open to other options to remove the word "we"